### DISCUSSION TITLE
v1.7.4

### DISCUSSION THREAD

**Discussion by aluminum-ice** [aluminum-ice/pwnagotchi#14] [Announcements]
- Added support for the RPi3B+ and RPi4 via firmware patch bcm43455c0/7_45_206

<hr /><em>This discussion was created from the release <a href='https://github.com/aluminum-ice/pwnagotchi/releases/tag/v1.7.4'>v1.7.4</a>.</em>

**Comment by do-ki** [comment]
install this image in Raspberry Pi 3A+ I got blind bug shows up in less than 10 mins

Details posted here:
(https://www.reddit.com/r/pwnagotchi/comments/15dyosj/comment/juh3r8t/?utm_source=share&utm_medium=web2x&context=3)

Tried the v1.6.2 image with also the same firmware patch and it works without any issues.

**Comment by ZexX-pt** [comment]
Hello!

Has anyone tried the patch below to add support for Waveshare 2.13 V4 hat e-ink display?

https://github.com/evilsocket/pwnagotchi/pull/1135/commits/28c0adbb08ca5fcc6c36dc11d00f666eaecdc4cb

**Comment by kolesnikilya** [comment]
An error occurs on my raspberry 3b+ (waiting for bettercap API to be available ...), which I don't understand how to solve. Maybe you know what can help me?

**Comment by TenThousand1** [comment]
Trying version 1.7.5 on a Raspberry Pi Zero 2 W (RPiZ2W). It is probably a bit older as I was only able to find one of the 2's with an expensive kit of stuff I didn't really need. 

So far the only thing I have noticed is that the webUI seems to not reboot into auto mode, rebooted fine and reconnected, but was still in manual mode.

Still testing. Bluetooth seems to work fine.
